Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -110,3 +121,27 @@
     transferAmount: 15
   - type: InjectableSolution
     solution: ammo
+
+- type: entity
+  id: PelletShotgunFlare
+  name: pellet (.50 flare)
+  abstract: true
+  parent: BulletBase
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Projectiles/flare_buckshot.rsi
+    state: base
+  - type: Projectile
+    damage:
+      groups:
+        Burn: 20
+  - type: PointLight
+    enabled: true   
+    color: "#FF8080"
+    radius: 15.0
+    energy: 9.0
+  - type: Flammable
```

Comment:
Probably not then, do you know what component I would use for spreading fires without the other stuff flammable component has, or does that not exist yet?

Reply:
Have you tested this in game against a spawned mob to see if it lights them on fire? Not sure if your intended behavior works with how we do fire currently. If you want it to light atmos on fire as it passes through you need something that exposes to atmos (hotspot). Best consulting Vera on how to go about this so that we have both things.

Imo firing guns should also light fires but that's something else.

Reply:
i imagine you can just make the component yourself. shouldnt be hard